.github/workflows/stale.yml

```diff
@@ -8,6 +8,10 @@ on:
 permissions:
   issues: write
 
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
```

Comment:
```suggestion
jobs:
```
I don't think this is necessary, as stale only runs once a day.

Reply:
This and `tidelift.yml` can also be triggered manually via `workflow_dispatch:`, so potentially could have more than one in the queue at once.

But I've no strong opinion either way.